### PR TITLE
DOCSP-24434: Add meta descriptions

### PR DESCRIPTION
**Pull request by ccho-mongodb**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-node/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-24434>
Staging - 
https://docs-mongodbcom-staging.corp.mongodb.com/node/docsworker-xlarge/DOCSP-24434-meta/usage-examples/find-operations/

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [ ] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working?

### CODE REVIEW

File: source/fundamentals/crud/read-operations.txt [hunk 1]
```diff
@@ -2,7 +2,9 @@
 Read Operations
 ===============
 
-.. default-domain:: mongodb
+.. meta::
+   :description: Learn the concepts and commands for MongoDB read operations by using the {+driver-long+}
+related to retrieving data from MongoDB by using the {+driver-long+}.
```
Comment (rustagir) [inline source/fundamentals/crud/read-operations.txt:7]: I: this seems like a run-on sentence. The driver is mentioned twice, which I think isn't necessary.

File: source/fundamentals/crud/write-operations.txt [hunk 2]
```diff
@@ -2,7 +2,8 @@
 Write Operations
 ================
 
-.. default-domain:: mongodb
+.. meta::
+   :description: Learn the concepts and commands for MongoDB write operations by using the {+driver-long+}.
```
Comment (rustagir) [inline source/fundamentals/crud/write-operations.txt:6]: S:
```suggestion
   :description: Learn the concepts and commands for executing MongoDB write operations by using the {+driver-long+}.
```

File: source/fundamentals/crud/read-operations.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (ccho-mongodb) [reply source/fundamentals/crud/read-operations.txt:7]: Thanks for spotting this, I think this is copypasta and needs to be deleted.

File: source/fundamentals/crud/write-operations.txt (hunk 2 above)
Comment (ccho-mongodb) [reply source/fundamentals/crud/write-operations.txt:6]: I considered "running" while formulating the sentence. However, parallelism can be tricky. While "executing" agrees with "commands", it doesn't agree with "concepts". And this form: "Learn the concepts of and commands for executing" unfortunately isn't proper grammar either.
Maybe "Learn about the commands for running ..." might be a good tradeoff. What do you think?

**Review by rustagir — APPROVED** [review]
lgtm

**Pull request merged**